docs/guides/setup/configuring_metadata_stores/how_to_configure_a_metricsstore.md

```diff
@@ -5,7 +5,7 @@ import TechnicalTag from '/docs/term_tags/_tag.mdx';
 
 Saving <TechnicalTag tag="metric" text="Metrics" /> during <TechnicalTag tag="validation" text="Validation" /> makes it easy to construct a new data series based on observed dataset characteristics computed by Great Expectations. That data series can serve as the source for a dashboard or overall data quality metrics, for example.
 
-Storing metrics is still a **beta** feature of Great Expectations, and we expect configuration and capability to evolve rapidly.
+Storing metrics is still a **experimental** feature of Great Expectations, and we expect configuration and capability to evolve rapidly.
```

Comment:
```suggestion
Storing metrics is still an **experimental** feature of Great Expectations, and we expect configuration and capability to evolve rapidly.
```

Minor typo, otherwise it's good to go.